pkg/agent/openflow/pipeline.go

```diff
@@ -873,6 +889,53 @@ func (c *client) arpNormalFlow(category cookie.Category) binding.Flow {
 		Done()
 }
 
+func (c *client) allowRulesMetricFlows(conjunctionID uint32, ingress bool) []binding.Flow {
+	metricTableID := IngressMetricTable
+	offset := 0
+	// We use the 0..31 bits of the ct_label to store the ingress rule ID and use the 32..63 bits to store the
+	// egress rule ID.
+	labelRange := binding.Range{0, 31}
+	if !ingress {
+		metricTableID = EgressMetricTable
+		offset = 32
+		labelRange = binding.Range{32, 63}
```

Comment:
declaring labelRange as two global variables to avoid repeated allocation, for example low32Range, high32Range?

Reply:
Updated.

Reply:
Updated.